bundles/io/org.openhab.io.caldav/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,53 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.io.caldav.internal
+Ignore-Package: org.openhab.io.caldav.internal
+Bundle-Name: openHAB CalDav Calendar
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.8.0.qualifier
+Bundle-Activator: org.openhab.io.caldav.internal.CalDavActivator
+Bundle-ManifestVersion: 2
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Description: This is the CalDav enhancement of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: com.google.common.collect;version="1.0.0",
+ org.apache.commons.codec;version="1.3.0",
+ org.apache.commons.codec.binary;version="1.4.0",
+ org.apache.commons.codec.net;version="1.4.0",
+ org.apache.commons.io;version="2.0.1",
+ org.apache.commons.lang,
+ org.apache.commons.lang.math,
+ org.apache.commons.logging;version="1.1.1",
+ org.joda.time,
+ org.joda.time.base,
+ org.joda.time.format,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.service,
+ org.openhab.core.types,
+ org.openhab.io.console,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.quartz,
+ org.quartz.impl,
+ org.quartz.impl.calendar,
+ org.quartz.impl.matchers,
+ org.quartz.utils,
+ org.slf4j
+Bundle-SymbolicName: org.openhab.io.caldav;singleton:=true
+Bundle-DocURL: http://www.openhab.org
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
+Service-Component: OSGI-INF/caldavloader.xml
+Bundle-ClassPath: .,
+ lib/commons-lang3-3.3.2.jar,
+ lib/httpclient-4.3.5.jar,
+ lib/httpcore-4.3.2.jar,
+ lib/ical4j-2.0-alpha1.jar,
```

Comment:
ical4j-2.0-beta1 is available since 22.08.2015, maybe upgrade? ( http://sourceforge.net/projects/ical4j/files/ical4j/ )